NiceIO.cs

```diff
@@ -758,11 +821,11 @@ static bool AlwaysTrue(NPath p)
 			return true;
 		}
 
-        private static bool IsLinux()
-        {
-            return Directory.Exists("/proc");
-        }
-    }
+		private static bool IsLinux()
```

Comment:
I that is fixing a block of space indented code.  I know it's ugly to include in this PR, but I thought it would be nice to clean it up